db/migrate/20170113225432_add_omniauth_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddOmniauthToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :provider, :string, null: false
```

Comment:
I am null falsing these columns and putting a unique index on them. 

This is assuming we will never have email/password authentication and having these columns on the user table prevents us from ever having multiple providers in the future (better approach is a has many association to an oauth identity table of some sorts if we want to future protect us). But with current requirements this should be fine.